### PR TITLE
Adding tests updated

### PR DESCRIPTION
**Pull request by EstherLerouzic**
Adding automatic tests with the code suggested by Jan Kundrat https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/20#discussion_r298215542 .

Will need to toggle travis ci on (I do have the settings button to put that on)

The test detected a type difference on flexi-n : probably a recent correction to account for negative values of n see #30

### CODE REVIEW

**Comment by EstherLerouzic** [comment]
@italobusi and @sergiobelotti could you please review the change and approve ? so that we can merge ?

File: tests/test_yang.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+base_name = 'https://raw.githubusercontent.com/YangModels/yang/master/'
+urls = ['standard/ietf/RFC/ietf-network@2018-02-26.yang',
+        'standard/ietf/RFC/ietf-yang-types@2010-09-24.yang',
+        'standard/ietf/RFC/ietf-network-topology@2018-02-26.yang',
+        'standard/ietf/RFC/ietf-inet-types@2010-09-24.yang',
+        'experimental/ietf-extracted-YANG-modules/ietf-te-types@2019-11-18.yang',
+        'experimental/ietf-extracted-YANG-modules/ietf-layer0-types@2019-11-28.yang',
+        'experimental/ietf-extracted-YANG-modules/ietf-te-topology@2019-02-07.yang']
+# TODO automatically retrieve list of versions based on listing in the github
```
Comment (italobusi) [inline tests/test_yang.py]: This piece of code needs to be maintained to get the latest version of the drafts until the automatic retrieval is implemented (TODO)

Shall we create a github open issue to track this point?

File: ietf-optical-impairment-topology.tree [hunk 2]
```diff
@@ -14,7 +14,7 @@ module: ietf-optical-impairment-topology
        +--ro media-channel-group* [i]
        |  +--ro i                 int16
        |  +--ro media-channels* [flexi-n]
-       |     +--ro flexi-n      uint16
+       |     +--ro flexi-n      int16
```
Comment (italobusi) [inline ietf-optical-impairment-topology.tree]: The proposed change is correct and aligned with the latest version of layer0-types. The value of n in G.694.1 is defined as "a positive or negative integer including 0" so it should be an int16 and not an uint16

File: tests/test_yang.py (hunk 1 above)
Comment (italobusi) [reply tests/test_yang.py:44]: As discussed during 2020-04-02 CCAMP call, I have created open issue #31

**Comment by EstherLerouzic** [comment]
replaced by #47

**Pull request closed without merging**